readme.txt

```diff
@@ -288,6 +273,9 @@ Minimum supported PHP version is 7.3.
 
 == Changelog ==
 
+### 2.0.2 (November XX, 2022) ###

```

Comment:
```suggestion
### 2.0.2 (November 16, 2022) ###

```

Reply:
I wasn't planning to deploy today because it's later in my day, and I wasn't sure when folks would get around to reviewing. If everything else looks good, though, I can change this to tomorrow's date and run through a release first thing.

Reply:
That's fine.  I'm resolving this.